WEB_API/app3.py

```diff
@@ -0,0 +1,97 @@
+import fnmatch
+import os
+import uuid
+import json
+import time
+from datetime import datetime
+from dotenv import load_dotenv
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from flask import Flask, request, jsonify, send_from_directory, abort, render_template
+from DB.database import User, Upload  # Import the User and Upload classes from your database module
+
+load_dotenv()

```

Comment:
Expressions outside of a function are a bad habit. This code will be called every time the file is loaded and imported.   Read more here: https://www.geeksforgeeks.org/python-main-function/amp/